docs/content/en/docs/workflows/ci-cd.md

```diff
@@ -4,20 +4,26 @@ linkTitle: "Continuous Delivery"
 weight: 40
 ---
 
-Skaffold offers several sub-commands for its workflows that make it quite flexible when integrating with CI/CD pipelines.
+Skaffold provides several features and sub-command "building blocks" that make it very useful for integrating with (or creating entirely new) CI/CD pipelines.
+The ability to use the same skaffold.yaml for iterative development and continuous delivery make handing off an application from a development team to an ops team a breeze!
 
+Let's start with the simplest use case: a single, full deployment of your application.
 
 ## `skaffold run`
+{{< maturity "run" >}}
 
-`skaffold run` is a single command for a one-off deployment. It includes all the following phases as it builds, tags, deploys and waits for the deployment to succeed if specified.
-We recommend `skaffold run` for a simple Continuous Delivery setup, where it is sufficient to have a single step that deploys from version control to a cluster.
-For more sophisticated Continuous Delivery pipelines, Skaffold offers building blocks that are described next:
+`skaffold run` is a single command for a one-off deployment. It runs through every major phase of the Skaffold lifecycle: building your application images, tagging these images (and optionally pushing them to a remote registry), deploying your application to the target cluster, and monitoring the created resources for readiness.
+
+We recommend `skaffold run` for the simplest Continuous Delivery setup, where it is sufficient to have a single step that deploys from version control to a cluster.
+
+For more sophisticated Continuous Delivery pipelines, Skaffold offers building blocks:
 
 - [healthcheck]({{<relref "/docs/workflows/ci-cd#waiting-for-skaffold-deployments-using-healthcheck">}}) - 
 wait for `deployments` to stabilize and succeed only if all deployments are successful
 - [`skaffold build`]({{<relref "/docs/workflows/ci-cd#skaffold-build-skaffold-deploy">}}) - build, tag and push artifacts to a registry
 - [`skaffold deploy`]({{<relref "/docs/workflows/ci-cd#skaffold-build-skaffold-deploy">}})  - deploy built artifacts to a cluster
 - [`skaffold render`]({{<relref "/docs/workflows/ci-cd#skaffold-render">}})  - export the transformed Kubernetes manifests for GitOps workflows
+<!-- - [`skaffold apply`]() - send hydrated Kubernetes manifests to the API server to create resources on the target cluster -->
```

Comment:
Should this be commented?  `apply` is mentioned several places below.

Reply:
oof, sorry. I was having trouble with hugo not being able to find this reference, let me fix